pact/updateRecall.pact.test.ts

```diff
@@ -27,7 +27,8 @@ pactWith({ consumer: 'manage-recalls-ui', provider: 'manage-recalls-api' }, prov
         contrabandDetail: 'Likely to bring in contraband because...',
         lastReleaseDateTime: '2020-08-03T00:00:00.000Z',
         lastReleasePrison: 'BAI',
-        localPoliceForce: 'Essex',
+        localPoliceForce: 'Dyfed-Powys Police',
```

Comment:
We update the label as well?

Reply:
Yes at this step @jonnywyatt .  Next step is for the API to always use only the ID.  After that we will remove the label/name string from the contract.